File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/upload/UploadRestForm.java
package com.isystk.sample.web.front.controller.api.v1.upload;

import com.isystk.sample.web.base.controller.html.BaseForm;
import org.springframework.web.multipart.MultipartFile;

public class UploadRestForm extends BaseForm {

  private static final long serialVersionUID = 7593564324192730932L;

  Integer imageId;

  MultipartFile imageFile;

  public Integer getImageId() {
    return this.imageId;
  }

  public MultipartFile getImageFile() {
    return this.imageFile;
  }

  public void setImageId(Integer imageId) {
    this.imageId = imageId;
  }

  public void setImageFile(MultipartFile imageFile) {
    this.imageFile = imageFile;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/upload/UploadRestForm.kt
package com.isystk.sample.web.front.controller.api.v1.upload

import com.isystk.sample.web.base.controller.html.BaseForm
import lombok.Getter
import lombok.Setter
import org.springframework.web.multipart.MultipartFile

@Setter
@Getter
class UploadRestForm : BaseForm() {
    var imageId: Int? = null
    var imageFile: MultipartFile? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}